frame/system/extentions/extentions.go

```diff
@@ -0,0 +1,100 @@
+package system
```

Comment:
Shouldn't the folder be named `extensions`?

Reply:
maybe, I'll change it